### ISSUE TITLE
Doesnt correctly work with themes

### ISSUE THREAD

**Issue by MiniSkyDiver** [denysdovhan/vacuum-card#394]
After cleaning cache in browser i see card with normal state:

![изображение](https://user-images.githubusercontent.com/48034802/169985910-b329612a-6d89-4a20-a487-7613b4a34008.png)

But after refreshing page it broking:

![изображение](https://user-images.githubusercontent.com/48034802/169986389-d67b0774-335c-46f1-b088-9baab413dede.png)

after couple refreshing with cleaning cache it fixing but broking after first refresh.

same situation on mobile app.

Maybe i have a problem with my configuration? Re-install doesnt helped me

**Versions:**

- Release: 2.6.3
- Browser: Firefox
- HomeAssistant version: 2022.5.5
- HomeAssistant for Android 03.14.2022

**Comment by denysdovhan** [comment]
This seems like a caching problem. Does this issue still persists?

**Comment by MiniSkyDiver** [comment]
> Does this issue still persists?

yup, issue still persists. The strangest thing is that the problem reoccurs after refreshing the page without clearing the cache. Same problem in mobile app confuses me.
